notificationworkerlambda/src/main/scala/com/gu/notifications/worker/NotificationWorkerLocalRun.scala

```diff
@@ -44,13 +48,26 @@ object NotificationWorkerLocalRun extends App {
     val sqsMessage = new SQSMessage()
     sqsMessage.setBody(Json.stringify(Json.toJson(tokens)))
     sqsMessage.setAttributes((Map("SentTimestamp" -> s"${Instant.now.toEpochMilli}").asJava))
-    event.setRecords(List(sqsMessage).asJava)
+    event.setRecords(List.fill(4)(sqsMessage).asJava)
     event
   }
+  val localTestContext: Context = new Context{
```

Comment:
I wonder whether a mock might be more appropriate here if we only need to have one of the variables defined?

Reply:
If we want to use mock objects, we may need to use the `spec2` and extend the class to `Mockito` (correct me if I was wrong), but this `NotificationWorkerLocalRun` is not a test case.  It is more of a command line Java entry class for us to run the Android sender lambda code locally.

Reply:
Oh I'm sorry, my mistake, I thought this was a test file. Yeah makes sense for a real invocation